pkg/resource/queue/sdk.go

```diff
@@ -223,6 +242,13 @@ func (rm *resourceManager) sdkUpdate(
 	latest *resource,
 	delta *ackcompare.Delta,
 ) (*resource, error) {
+	var err error
```

Comment:
Can we change the codegen such that this variable is declared in the return value part of the function block, one line above?

Reply:
sure, I can do that in a future PR @jljaco